cypress/integration/lesson14.spec.js

```diff
@@ -0,0 +1,49 @@
+import { faker } from '@faker-js/faker';
+
+const FAKE_FULL_NAME = faker.name.findName();
+const FAKE_EMAIL = faker.internet.email();
+const FAKE_ADDRESS = faker.address.secondaryAddress();
+const FAKE_PHONE = faker.phone.phoneNumber();
+
+describe('Lesson14: Text Box Form', () => {
+  before(() => {
+    cy.visit('https://demoqa.com/text-box');
+  });
+
+  afterEach(() => {
+    cy.reload(); // cleanup text from the inputs fields
```

Comment:
We can use "before" commands like "cy.clearLocalStorage() and cy.clearCookies()"